ruzm12-sp

maruzek opened the pull request:
Odevzdávám SP1 před obhajobou
Odkaz: https://eso.vse.cz/~ruzm12/sp1/
Můj repozitář: https://github.com/martinruzekk/buldok-ticketing

maruzek committed:
ruzm12-sp

Odevzdávám SP1 před obhajobou
Odkaz: https://eso.vse.cz/~ruzm12/sp1/
Můj repozitář: https://github.com/martinruzekk/buldok-ticketing

nvbach91 reviewed (changes requested):
- odevzdat do puvodniho Pull Requestu, nevytvaret novy
- jak se dostanu do administrace k vytvoreni zapasu
- nic se nestane kdyz kliknu na tlacitko vstupenky
- proc je datum v anglictine
- pozadavek na heslo 6 znaku neni dostatecne bezpecne

maruzek committed:
ruzm12-sp-final

Odevzdávám finální verzi po obhajobě.
Aplikace bude přístupná na https://eso.vse.cz/~ruzm12/sp1/ nebo https://eso.vse.cz/~ruzm12/sp2/
Můj repo: https://github.com/martinruzekk/buldok-ticketing

maruzek committed:
ruzm12-final-2

Odevzdávám finální verzi po obhajobě
Aplikace bude dostupná na https://eso.vse.cz/~ruzm12/sp1/ nebo na https://eso.vse.cz/~ruzm12/sp2/
Můj repo: https://github.com/martinruzekk/buldok-ticketing

maruzek committed:
ruzm12-sp-db

Přidávám export databáze
Aplikace je již dostupná na https://eso.vse.cz/~ruzm12/sp1/
Můj repo: https://github.com/martinruzekk/buldok-ticketing
